src/primitives.js

```diff
@@ -1,3 +1,18 @@
+// This file contains implmentations using three.js of Mathematica and
+// Mathics Graphics3D primitives like "Sphere", or "Cuboid, etc.
```

Comment:
The comment style is:
```
// phrase 1. Phrase 2 without the dot in the end

// the line above contains no "//"

Reply:
I tried to keep that in things that weren't changed.  To me this is a bit idiosyncratic and draws attention. And in some places the lack of punctuation at the end of a sentence sometimes is confusing because it is not clear where the sentence end.

There is a series of stories about a cockroach who write poetry and has to leap on the keys of a typewrite, and since typing each letter,he leaves of the sentence endings and can't work this shift key because he can only jump on one key at a time. In those stores, the jumping is meant to be amusing. You are not a cockroach are you? (On the Internet, one never knows). 

But anyway, then this is your code. Please go back on remove punctuation to fit your style.

Reply:
> And in some places the lack of punctuation at the end of a sentence sometimes is confusing because it is not clear where the sentence end.

Ok, I'll change my comments, I just like to keep the same style over all the code.